<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomeworkManagerUI</class>
 <widget class="QWidget" name="HomeworkManagerUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>–ú–µ–Ω–µ–¥–∂–µ—Ä –¥–æ–º–∞—à–Ω–∏—Ö –∑–∞–¥–∞–Ω–∏–π</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="addTaskButton">
     <property name="text">
      <string>–î–æ–±–∞–≤–∏—Ç—å –∑–∞–¥–∞–Ω–∏–µ</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="taskList"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="deleteButton">
       <property name="text">
        <string>üóëÔ∏è –£–¥–∞–ª–∏—Ç—å</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="completeButton">
       <property name="text">
        <string>‚úÖ –í—ã–ø–æ–ª–Ω–∏—Ç—å</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="completedLabel">
     <property name="text">
      <string>–í—ã–ø–æ–ª–Ω–µ–Ω–Ω—ã–µ –∑–∞–¥–∞–Ω–∏—è:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="completedTaskList"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
